demo/components/victory-chart-demo.js

```diff
@@ -386,7 +386,9 @@ class App extends React.Component {
 
           <VictoryChart style={chartStyle} scale={"linear"}>
             <VictoryAxis/>
-            <VictoryAxis label="AWESOME LABEL" dependentAxis crossAxis={false} offsetX={30}/>
+            <VictoryAxis label="AWESOME LABEL" dependentAxis crossAxis={false} offsetX={30}
+              style={{axisLabel: {padding: 55}}}
```

Comment:
@boygirl yeah, I was having a hard time figuring out a better way for when there's an offset... maybe checking to see if there's an offset in props and then just adding that to the base number